examples/images_contours_and_fields/interpolation_none_vs_nearest.py

```diff
@@ -1,62 +0,0 @@
-"""
```

Comment:
It does point out an important difference between the "none" and "nearest" methods though.  This information needs to remain available somewhere.